### ISSUE TITLE
scroller and fix-column like jquery datatable

### ISSUE THREAD

**Issue by eafshary** [ratiw/vuetable-2#211]
Is there any possibility to have fix-column or scroll loading on this component?

**Comment by ratiw** [comment]
@eafshary I don't think so. Vuetable was implemented to make use of standard HTML table to take advantage of CSS framework styling. Datatable on the other hand uses jquery to convert underlying html table to div to allow those functionality.
